spec/acceptance/bootstrap_rpm_spec.rb

```diff
@@ -1,6 +1,10 @@
 require 'spec_helper_acceptance'
 
-describe 'bootstrap_rpm' do
+describe 'bootstrap_rpm', :order => :defined do
+
+  before(:suite) do
+    `rm -rf /var/www/html/pub/*rpm`
```

Comment:
This executes locally, not on the remote server. You can verify this by running it twice and you'll see it fails:
```
BEAKER_destroy=no be rspec spec/acceptance/bootstrap_rpm_spec.rb
BEAKER_provision=no BEAKER_destroy=no be rspec spec/acceptance/bootstrap_rpm_spec.rb
```

The second run reuses the machine and its state, which makes iterating on a change much faster.

This makes it pass for me:

```suggestion
  before(:all) do
    on hosts, 'rm -rf /var/www/html/pub/*rpm'
```